docs/starter-web.md

```diff
@@ -5,28 +5,30 @@ core concepts.
+| `cors.allowed-origin-patterns` _string_            | Comma separated list of URL patterns for which CORS requests are allowed.                                                                                                                                                                                                                                   | _none allowed_                                                                                                                                           | `https://*.all-subdomains.com, https://static-domain.com`    | `CORS_ALLOWEDORIGINPATTERNS`              |
+| `request.body.max.size` _size_                     | The maximum size allowed for request body data sent by a client.                                                                                                                                                                                                                                            | _1 MB_                                                                                                                                                   | `100 KB`, `10MB`                                             | `REQUEST_BODY_MAX_SIZE`                   |
+| `enable.json.logging` _boolean_                    | If logs should be printed as JSON. _Note: This config param is not available for application.properties or application.yaml_                                                                                                                                                                                | _false_                                                                                                                                                  | `true`                                                       | `ENABLE_JSON_LOGGING`                     |
+| `management.otlp.tracing.endpoint` _string_        | Base url to OTLP Collector instance. It applies for http or gRPC, if enabled                                                                                                                                                                                                                                | `http://grafana-agent-traces.monitoring:4317`                                                                                                            | `http://localhost:4318`                                      | `MANAGEMENT_OTLP_TRACING_ENDPOINT`        |
```

Comment:
I would prefer to add quotes to all string defaults/examples instead of removing them here. That would actually clarify my question regarding _none_ which I see answered in the defaults. But the reader does not see them as prominent as in the PR.